code/modules/background/space_sectors/space_sector.dm

```diff
@@ -12,6 +12,8 @@
 	var/list/possible_exoplanets = list(/obj/effect/overmap/visitable/sector/exoplanet/snow, /obj/effect/overmap/visitable/sector/exoplanet/desert)
 	///Guaranteed planets to spawn. This ignores the map exoplanet limit, so don't put too many planets in here.
 	var/list/guaranteed_exoplanets = list()
+	///Banned exoplanets. These will be removed from the possible exoplanets list.
+	var/list/banned_exoplanets = list()
```

Comment:
document what this list is supposed to contain
(types like `/obj/effect/overmap/visitable/sector/exoplanet/...`)